src/Equinor.ProCoSys.IPO.Email/Equinor.ProCoSys.IPO.Email.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I suggest not creating a separate project for this, and rather have the service in existing project structure